vispy/visuals/bar.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+from .mesh import MeshVisual
+
+
+class BarVisual(MeshVisual):
+    """Visual that calculates and displays a bar graph
+
+    Parameters
+    ----------
+    index : array-like
+        Index of bar
+    bottom : int | float | array-like
+        Bottom of the bars
+    height : int | float | array-like
+        Height of the bars
+    width : int | float
+        Width of all bars
+    color : instance of Color
+        Color of the histogram.
+    """
+
+    def __init__(self, color='w', index=None, bottom=None, height=None, width=0.8):
```

Comment:
It looks like these defaults are never handled for `index`, `bottom`, or `height`. Should these be positional arguments? Or have different defaults?

Reply:
Well index and height would have to be supplied to graph the bars, and as suggested bottom should probely 0 as default